frontend/src/features/settings/components/creditAccount/StatusChip.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  CreditAccountStatusType,
+  CreditAccountStatusDisplayValues,
+} from "../../types/creditAccount";
+import "./StatusChip.scss";
+
+interface StatusChipProps {
+  status: CreditAccountStatusType | "SUSPENDED";
+}
+
+export const StatusChip = ({ status }: StatusChipProps) => {
+  if (status === "ACTIVE") {
+    return;
+  }
+  if (status === "SUSPENDED") {
+    return (
+      <span role="status" className="status-chip status-chip--suspended">
+        Suspended
+      </span>
+    );
+  }
+  return (
+    <span
+      role="status"
+      className={`status-chip status-chip--${CreditAccountStatusDisplayValues[status].toLowerCase().replace(" ", "-")}`}
```

Comment:
Extract out the status display string part into a variable, and replace this long className with that variable